tests/countries/test_suriname.py

```diff
@@ -0,0 +1,312 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries import Suriname, SR, SUR
+from tests.common import CommonCountryTests
+
+
+class TestSuriname(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1972, 2050)
+        super().setUpClass(Suriname, years=years, years_non_observed=years)
+
+    def test_country_aliases(self):
+        self.assertAliases(Suriname, SR, SUR)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Suriname(years=1971))
+
+    def test_new_years_day(self):
+        name = "Nieuwjaarsdag"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1972, 2050)))
+        dt = (
+            "2028-01-03",
+            "2033-01-03",
+            "2034-01-02",
+        )
+        self.assertHolidayName(f"Dag na de {name}", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_day_of_liberation_and_renewal(self):
+        name = "Dag van Bevrijding en Venieuwing"
+        self.assertHolidayName(name, (f"{year}-02-25" for year in range(1981, 1993)))
+        self.assertNoHolidayName(name, (f"{year}-02-25" for year in range(1993, 2012)))
+        self.assertHolidayName(name, (f"{year}-02-25" for year in range(2012, 2021)))
+        self.assertNoHolidayName(name, (f"{year}-02-25" for year in range(2021, 2050)))
```

Comment:
```suggestion
        self.assertHolidayName(
            name, (f"{year}-02-25" for year in (*range(1981, 1993), *range(2012, 2021)))
        )
        self.assertNoHolidayName(name,  range(1993, 2012), range(2021, 2050))
```
Such check (by years range) is more strict than check with a specific date. Sometimes it's possible to add laso
```python
        self.assertNoHoliday(f"{year}-02-25" for year in (*range(1993, 2012), *range(2021, 2050)))
```
(when in some years some other movable holiday doesn't fall on this date)